packages/code-du-travail-frontend/src/search/search.service.js

```diff
@@ -6,20 +6,19 @@ const {
   publicRuntimeConfig: { API_URL, SUGGEST_URL }
 } = getConfig();
 
-const fetchResults = endpoint => async (query = "", excludeSources = "") => {
-  const url = `${API_URL}/${endpoint}?q=${encodeURIComponent(
+const fetchSearchResults = async ({ query = "", excludeSources = "" }) => {
```

Comment:
you don't need excluded sources

Reply:
I use it here: https://github.com/SocialGouv/code-du-travail-numerique/pull/1272/files#diff-5dfa1a6c4cb16654a1db037370ffacc0R57